epub34/authoring/index.html

```diff
@@ -3116,44 +3116,69 @@ <h3>Vocabulary association mechanisms</h3>
-						referenced from the default vocabularies do not include a prefix as the mapping reading systems
-						use to map to a URL is predefined.</p>
-
-					<p>The power of the <var>property</var> data type lies in its easy extensibility. To incorporate new
-						terms and properties, it is only necessary to declare a <a href="#sec-prefix-attr">prefix</a>.
-						In another authoring convenience, this specification also <a
-							href="#sec-metadata-reserved-prefixes">reserves prefixes</a> for many commonly used
-						publishing vocabularies (i.e., their declaration is not required).</p>
-
-					<p>The following sections provide additional details on the <var>property</var> data type and
-						vocabulary association mechanism.</p>
+					<p>The package document allows metadata expressions, and metadata extensibility, through the use of
+						[=compact URLs=]. A compact URL is very similar to a CURIE [[rdfa-core]] in how they represent
+						URLs [[url]] — both expressions consist of an optional prefix followed by a required
+						reference.</p>
+
+					<p>The prefix, whether literal or implied, is a shorthand mapping of a URL that typically resolves
+						to a vocabulary, such as Dublin Core [[dcterms]] or Schema.org [[schema-org]]. For example, the
+						prefix <code>dcterms</code> is used instead of having to add the full URL
+							<code>http://purl.org/dc/terms/</code> every time. Using prefixes makes authoring metadata
+						less error prone while still retaining the uniqueness of the vocabulary being used.</p>
+
+					<p>The reference is the specific term or property being declared. It is separated from its prefix by
+						a colon. A prefix is not always required for a compact URL because of what are called [=default
+						vocabularies=]. Each attribute that accepts a compact URL also defines a default vocabulary of
+						terms or properties. The absence of a prefix means the compact URL represents a value from the
+						default vocabulary.</p>
+
+					<aside class="example" title="Types of compact URLs">
+						<p>In this example, the compact URL in the <code>property</code> attribute has both a prefix
+								(<code>dcterms</code>) and a reference (<code>modified</code>).</p>
+
+						<pre>&lt;meta property="dcterms:modified"&gt;2025-10-18T12:00:00Z&lt;/meta&gt;</pre>
+
+						<p>In this example, the compact URL in the <code>properties</code> attribute is taken from its
+							default vocabulary so only the reference (<code>cover-image</code>) is present:</p>
+
+						<pre>&lt;item href="images/cover.jpg" properties="cover-image" media-type="image/jpeg" /&gt;</pre>
+					</aside>
+
+					<p>When a [=reading system=] converts a compact URL to a full URL, the result normally resolves to a
+						fragment within that vocabulary that contains human- and/or machine-readable information about
+						the term, but this is not required. Prefixes are ultimately just unique identifiers, similar to
```

Comment:
```suggestion
					<p>When a [=reading system=] converts a compact URL to a full URL, the result usually resolves to a
						fragment within that vocabulary that contains human- and/or machine-readable information about
						the term, but this is not required. Prefixes are ultimately just unique identifiers, similar to
```

"normally" suggests (for me) that other usages are "abnormal". This isn't the case; there has been a long-standing battle whether terms should be fragments or URL paths. The best known (for us) case for the latter is dcterms, which is surely not "abnormal"...

Reply:
Alternatively, we could also say "normally resolves to fragment or a path within that vocabulary". In that case "normally" is fine: it is possible to end the prefix with a '?', for example, or without any of the '#', '/', or '?' characters, but that would indeed be "abnormal".

Reply:
Prefixes have to be valid to xsd:anyURI, so what you get when you combine that with the reference can't really be predicted. I think "usually" or "typically" is fine here, since all we're saying is that the most common result is a URL that points directly to the definition of the term or property being used. For an intro, we should probably avoid getting too deep into the weeds.

Reply:
Ok. Let us leave it at "usually"!